### PR TITLE
Add repair feature to wiki

### PR DESCRIPTION
**Pull request by PabstMirror**

### CODE REVIEW

File: docs/wiki/feature/repair.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+---
+
+## 1. Overview
+
+Adds a more realistic repair system.
+- Replacing damaged wheels and tracks requires replacement items (using <a href="{{ site.baseurl }}/wiki/framework/cargo-framework.html">ace_cargo</a>.
+- Can repair specific hitpoints on vehicles.
+- Can designate buildings and/or vehicles as repair zones allowing the ability to do a full repair.
+
+## 2. Repairing
```
Comment (bux) [inline docs/wiki/feature/repair.md]: need a blank line below each header

File: docs/wiki/feature/repair.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## 3. Object Variables / Configs
+
+### 3.1 Unit's Repair Training
+- Determines if a unit can repair damaged hitpoints and if so, how much can be restored.
+- Can easily be set via 3den attribute.
+
+SetVar: `"ACE_IsEngineer"` - number 0 - 2 (can also be true/false)
+- 0 / false: Untrained (by default settings will only be able to replace wheels)
+- 1 / true: Engineer
+- 2: Specialist (allowing a higher level of repairs, based on settings) (by default settings is the only one with access to full repair)
```
Comment (Phyma) [inline docs/wiki/feature/repair.md]: New translation of the repair guys.

`2: Advanced Engineer (allowing a higher level of repairs, based on settings) (by default settings is the only one with access to full repair)`

**Comment by Phyma** [comment]
If this get accepted the PR #5248 should also be accepted. Because this wiki uses Advanced Engineer that is introduced with the other PR.

File: docs/wiki/feature/repair.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- Drag the spare next to the wheel/track that needs replacing
+- Remove the damaged wheel (not needed for track repair)
+- Install new wheel or track
+
+### 2.2 Repairing hitpoints
+
+- Amount repaired is based on settings and training level
+- Using the interaction system select the hitpoint to repair
+
+## 3. Object Variables / Configs
```
Comment (jonpas) [inline docs/wiki/feature/repair.md]: This would ideally go to a repair-framework page.

File: docs/wiki/feature/repair.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+version:
+  major: 3
+  minor: 3
+  patch: 0
+---
+
+## 1. Overview
+
+Adds a more realistic repair system.
+- Replacing damaged wheels and tracks requires replacement items (using <a href="{{ site.baseurl }}/wiki/framework/cargo-framework.html">ace_cargo</a>.
```
Comment (jonpas) [inline docs/wiki/feature/repair.md]: Can use markdown: `[Cargo System]({{ site.baseurl }}/wiki/framework/cargo-framework.html)`.

**Pull request merged**